tests/test_order.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+import rtoml
+
+"""
+When testing preserving of order we just check if
```

Comment:
This is not the right place for a description, instead add it as a docstring to the test function